remoteworkitem/remoteworkitem.go

```diff
@@ -11,16 +11,47 @@ const (
 	SystemTitle        = "system.title"
 	SystemDescription  = "system.description"
 	SystemStatus       = "system.status"
-	ProviderGithub     = "github"
-	ProviderJira       = "jira"
+	SystemAssignee     = "system.assignee"
+	SystemCreator      = "system.creator"
+
+	// The keys in the flattened response JSON of a typical Github issue.
+
+	GithubTitle       = "title"
+	GithubDescription = "body"
+	GithubState       = "state"
+	GithubId          = "id"
+	GithubCreator     = "user.login"
+	GithubAssignee    = "assignee.login"
+
+	// The keys in the flattened response JSON of a typical Jira issue.
+
+	JiraTitle    = "fields.summary"
+	JiraBody     = "fields.description"
+	JiraState    = "fields.status.name"
+	JiraId       = "self"
+	JiraCreator  = "fields.creator.key"
+	JiraAssignee = "fields.assignee"
+
+	ProviderGithub = "github"
+	ProviderJira   = "jira"
 )
 
 var WorkItemKeyMaps = map[string]WorkItemMap{
 	ProviderGithub: WorkItemMap{
-		AttributeExpression("title"): SystemTitle,
-		AttributeExpression("body"):  SystemDescription,
-		AttributeExpression("state"): SystemStatus,
-		AttributeExpression("id"):    SystemRemoteItemId,
+		AttributeExpression(GithubTitle):       SystemTitle,
+		AttributeExpression(GithubDescription): SystemDescription,
+		AttributeExpression(GithubState):       SystemStatus,
+		AttributeExpression(GithubId):          SystemRemoteItemId,
+		AttributeExpression(GithubCreator):     SystemCreator,
+		AttributeExpression(GithubAssignee):    SystemAssignee,
+	},
+	ProviderJira: WorkItemMap{
+		AttributeExpression(JiraTitle):    SystemTitle,
+		AttributeExpression(JiraBody):     SystemDescription,
+		AttributeExpression(JiraState):    SystemStatus,
+		AttributeExpression(JiraId):       SystemRemoteItemId,
+		AttributeExpression(JiraCreator):  SystemCreator,
```

Comment:
We need to make a note that XCreator and XAssignee need to be mapped/created via the 'Account' service somehow to get the 'Real' value we should input here which is Login.ID